scripts/generate_tables_helper.sh

```diff
@@ -665,6 +702,6 @@ fetch_reference_proteomes() {
   local reference_proteome_url="https://rest.uniprot.org/proteomes/stream?fields=upid,organism_id,protein_count&format=tsv&query=(*)+AND+(proteome_type:1)"
 
   mkdir -p "$output_dir"
-  curl -s "$reference_proteome_url" | tail -n +2 | cat -n | sed 's/^ *//' | $CMD_LZ4 - > "$output_dir/reference_proteomes.tsv.lz4"
+  curl -s "$reference_proteome_url" | tail -n +2 | sort | $CMD_LZ4 - > "$output_dir/reference_proteomes.tsv.lz4"
```

Comment:
You can explicitly tell the `sort` command here that it only needs to look at the first column, this will potentially speed up the sorting even further.